Mathlib/GroupTheory/GroupAction/Primitive.lean

```diff
@@ -275,4 +290,113 @@ instance (priority := 100) IsPreprimitive.isQuasiPreprimitive [IsPreprimitive M
 
 end Normal
 
+section Finite
+
+namespace IsPreprimitive
+
+variable {M : Type*} [Group M] {α : Type*} [MulAction M α]
+variable {N β : Type*} [Group N] [MulAction N β]
```

Comment:
`#where` tells me that we already have `{G X : Type*} [Group G] [MulAction G X]` here so maybe we could reuse them instead of `M` and `α` and for consistency of naming:
```suggestion
variable {H Y : Type*} [Group H] [MulAction H Y]
```
for the roles of `N` and `β`